### PR TITLE
Requeue tasks mgmt command.

### PR DESCRIPTION
**Pull request by mklauber**

### CODE REVIEW

File: seo/management/commands/requeue_failures.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+from django.core.management.base import NoArgsCommand
+from djcelery.models import TaskState
+from datetime import datetime, timedelta
+from tasks import task_etl_to_solr
+
+class Command(NoArgsCommand):
+    args = 'None'
+    help = 'Checks for the existence of member logos on the content server'
```
Comment (tdphillips) [inline seo/management/commands/requeue_failures.py]: fry

**Comment by aspidites** [comment]
I don't like the 'None' and eval combo.

File: seo/management/commands/requeue_failures.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    def handle_noargs(self, **options):
+        eight_hours_ago = datetime.now() - timedelta(hours=8)
+
+        failed_tasks = TaskState.objects.filter(state__in=['FAILURE', 'STARTED', 'RETRY'], 
+                                                tstamp__gt=eight_hours_ago, 
+                                                name__in=['tasks.etl_to_solr',
+                                                          'tasks.priority_etl_to_solr'])
+
+        for task in failed_tasks:
+            task_etl_to_solr.delay(*eval(task.args))
```
Comment (aspidites) [inline seo/management/commands/requeue_failures.py]: I don't think this is doing what you want:

```python
>>> from __future__ import print_function
>>> print(*eval("None"))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: print() argument after * must be a sequence, not NoneType
>>> 
```

Comment (JLMcLaugh) [inline seo/management/commands/requeue_failures.py]: task.args is a string literal of the arguments passed into the original tasks. eval would split the string into 3 string arguments. But... this is really insecure. We should use celery's built in retry.

Comment (tdphillips) [inline seo/management/commands/requeue_failures.py]: @aspidites: The args attribute on Command is not the same as task.args.

Not saying this is great, but eval("None") isn't happening.

Comment (aspidites) [inline seo/management/commands/requeue_failures.py]: @tdphillips , oh, gotcha.

Comment (mklauber) [inline seo/management/commands/requeue_failures.py]: For what it's worth, `TaskState` doesn't have a retry method (It's a record of the task execution, not the task object itself).

Comment (mklauber) [inline seo/management/commands/requeue_failures.py]: But I may be able to try to parse and load the data via [`ast.literal_eval`](https://docs.python.org/2/library/ast.html#ast.literal%5Feval).

**Comment by aspidites** [comment]
If it wasn't already obvious, I'm good with this now. Anyone else?

**Pull request merged**